src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/ProfilerEditingTarget.java

```diff
@@ -528,6 +524,17 @@ public String getDefaultNamePrefix()
       return "Profile";
    }
    
+   private void clearProfileCache()
+   {
+      server_.clearProfile(htmlLocalPath_, new ServerRequestCallback<JavaScriptObject>()
```

Comment:
Does `htmlLocalPath_` need a null check? Since we call it on dismiss, IIRC exception here will keep the tab from closing.